server/src/main/kotlin/com/mobilejazz/colloc/CollocApplication.kt

```diff
@@ -43,6 +51,10 @@ class CollocController {
       .contentLength(bytes.size.toLong())
       .body(bytes)
   }
+
+  private fun injectParamsIntoAndroidLocalizationEncoder(id: String) {
+    beanFactory.getBean(AndroidLocalizationEncoder::class.java).importedUrlLocation = id.toFullImportedFileURL().toString()
+  }
```

Comment:
Not needed, you can remove it. 

Read message in the `LocalizationAndroidEncoder`